SConstruct

```diff
@@ -58,6 +59,7 @@ AddOption(
     '--qt',
     dest='qt',
     type='string',
+    default='/usr',
```

Comment:
Oups your are right I did not see that you clean the indentation this way. I've also done that at commit 164378e8fccf7336af85b9fe3360b5996b6dc15c